tozny/resource_realm.go

```diff
@@ -234,13 +259,30 @@ func resourceRealmUpdate(ctx context.Context, d *schema.ResourceData, m interfac
 		return diag.FromErr(err)
 	}
 
-	if d.HasChanges("mpc_enabled", "tozid_federation_enabled") {
+	if d.HasChanges("mpc_enabled", "tozid_federation_enabled", "forgot_password_custom_link", "forgot_password_custom_text") {
+
 		federation_setting := d.Get("tozid_federation_enabled").(bool)
 		mpc_setting := d.Get("mpc_enabled").(bool)
-		err = toznySDK.RealmSettingsUpdate(ctx, d.Get("realm_name").(string), identityClient.RealmSettingsUpdateRequest{
+
+		settingsUpdateRequest := identityClient.RealmSettingsUpdateRequest{
 			MPCEnabled:             &mpc_setting,
 			TozIDFederationEnabled: &federation_setting,
-		})
+		}
+		customForgetPasswordLink, exists := d.GetOk("forgot_password_custom_link")
+
+		if exists {
+			customForgetPasswordLinkStr := customForgetPasswordLink.(string)
+			settingsUpdateRequest.ForgotPasswordCustomLink = &customForgetPasswordLinkStr
+		}
+
+		customForgetPasswordText := d.Get("forgot_password_custom_text")
+
+		if exists {
+			customForgetPasswordTextStr := customForgetPasswordText.(string)
+			settingsUpdateRequest.ForgotPasswordCustomText = &customForgetPasswordTextStr
+		}
+
+		err = toznySDK.RealmSettingsUpdate(ctx, d.Get("realm_name").(string), settingsUpdateRequest)
 		if err != nil {
 			return diag.FromErr(err)
 		}
```

Comment:
make sure to add `d.Set("forgot_password_custom_text", d.Get("forgot_password_custom_text"))` and link